Sources/HttpPipeline/Response.swift

```diff
@@ -129,7 +129,7 @@ public struct Response {
     }
 
     public static func clearCookie(_ name: String) -> Header {
-      return .setCookie(name, "", [CookieOption.maxAge(0), .expires(0)])
```

Comment:
Since we were operating on `TimeInterval` here, Swift seemed confused without the qualifying `CookieOption`. Changing `expires` to take a `Date` prevents this ambiguity (which is prob better for the API). We also immediately convert to a `Date` anyway.